src/frontend/post-github-setup.ts

```diff
@@ -25,53 +26,51 @@ const renderGitHubSetupPageVersion = async (
 	res.render("github-setup.hbs", gitHubSetupPagePayload)
 };
 
-const validateJiraSite = (
-	jiraSiteUrl: string,
+const validateJiraSite = async (
+	jiraHost: string,
 	domain: string,
 	req: Request,
 	res: Response
-): void => {
-	// Check that the entered domain is valid by making a request to the status endpoint
-	axios(`${jiraSiteUrl}/status`, {
-		method: "GET",
-		headers: {
-			"content-type": "application/json",
-		},
-	})
-		.then((response) => {
-			// If Jira site is valid, response returns a state of RUNNING
-			if (response?.data?.state === "RUNNING") {
-				res.redirect(
-					req.session.githubToken
-						? getJiraMarketplaceUrl(jiraSiteUrl)
-						: "/github/login"
-				);
-			}
-		})
-		.catch((error) => {
-			// If Jira site is not valid, it returns a 404
-			req.log.error({ error }, "Invalid Jira site entered.");
-			renderGitHubSetupPageVersion(domain, req, res);
+): Promise<void> => {
+
+	// Check to see if we already have an installation of the app
+	if(await Installation.getForHost(jiraHost)) {
+		// If so, redirect to the app itself
+		return res.redirect(getJiraAppUrl(jiraHost));
+	}
```

Comment:
Seemed easier to find the install in our DB before doing a status check on jira.

Reply:
and if already installed, you can go directly to the app, no need to install it.

Reply:
Looks like a corner-case, should we maybe add a log to have visibility how often it happens?